common/logger.go

```diff
@@ -15,8 +16,19 @@ type Logger struct {
 	LogFile *os.File
 }
 
-func CreateLogger(logFile string) *Logger {
-	f, err := os.OpenFile(logFile, os.O_CREATE|os.O_WRONLY|os.O_APPEND, 0644)
+func CreateLogger(logPath string, logFile string) *Logger {
+	if _, err := os.Stat(logPath); os.IsNotExist(err) {
+		err := os.MkdirAll(logPath, 0755)
+		if err != nil {
+			// TODO: How to handle without a logger?
```

Comment:
I think we panic.
at this point if we expected creating a logger and can't do so we would like notify the user things does not go as expected.


maybe later we can return an EmptyLogger and some how notify user an err occurred (TODO)

Reply:
Fixed (probably)